Makefile.in

```diff
@@ -31,15 +31,23 @@ dune=@dune@
 # make, since it seems faster.
 #
 # CR-someday mshinwell: We should replace this with a coldstart using dune.
+#
+# The compiler can discard flambda-backend flags from OCAMLPARAM
+# by listing them in "ocaml/stdlib/ocaml_compiler_internal_params".
+# Copy this file to install directory of stage0 to make it available
+# to the build of the compiler itself in stage0 (instead of patching the
+# upstream compiler to look for it in the right place).
 .PHONY: stage0
-stage0: _build0/config.status
+stage0: _build0/config.status ocaml_compiler_internal_param
 	rm -f .rsync-output
 	rsync -i -a --filter=':- $$(pwd)/ocaml/.gitignore' \
 	  $$(pwd)/ocaml/ $$(pwd)/_build0 \
 	  | grep -v '/$$' \
 	  | tee .rsync-output
-	if [ -s .rsync-output ] || ! [ -d @stage0_prefix@ ]; then \
+	if [ -s .rsync-output ] || ! [ -d @stage0_prefix@/bin ]; then \
```

Comment:
Why is this change needed?

Reply:
yeah, I think so.. the action here creates @stage_prefix@/lib so it can copy ocaml_compiler_internal_params into it, even if install didn't run.